code/game/mecha/mech_fabricator.dm

```diff
@@ -62,11 +88,15 @@
 		T += Ml.rating
 	time_coeff = round(initial(time_coeff) - (initial(time_coeff)*(T))/5,0.01)
 
+	update_static_data(usr)
+
 /obj/machinery/mecha_part_fabricator/examine(mob/user)
 	. = ..()
 	if(in_range(user, src) || isobserver(user))
 		. += "<span class='notice'>The status display reads: Storing up to <b>[rmat.local_size]</b> material units.<br>Material consumption at <b>[component_coeff*100]%</b>.<br>Build time reduced by <b>[100-time_coeff*100]%</b>.</span>"
 
+
+// TODO - Delete me, I no longer work.

```

Comment:
Looks like I forgot my TODO sweep.

Removing the required access doesn't do anything (to my knowledge, I don't believe the current Exofab does anything when emagged either) - Rather than implement it, I had it slated for removal pending future ideas for more interesting emag interations. Turns out I never deleted it, duh-doy.